### ISSUE TITLE
Renaming of project "SCEditor" possible?

### ISSUE THREAD

**Issue by q2apro** [samclarke/SCEditor#649]
Every time I google for things about SCEditor, Google Search suggests "CKEditor". Pain in the back ... 

I guess the best way to overcome this is to rename the entire project.

@samclarke Is the planned/possible?

**Comment by samclarke** [comment]
Yeah unfortunately I didn't really think the name through very much when I named it (I'm bad at project names).

I'm not sure if it would be renaming it as this point but if someone has a good name suggestion it would be worth considering.
